### DISCUSSION TITLE
Resize utility (through code) and/or compressor utility

### DISCUSSION THREAD

**Discussion by darkylmnx** [advanced-cropper/vue-advanced-cropper#201] [Ideas]
Hi,

I was wondering if you were planning to have a resize utility that would have basic resize functions without the UI (resize to fixed dimensions, crop to fixed dimensions with position, resize to container dimensions, upscale, downscale...).

There are some situations where I don't need the use to see a UI, I just want to resize his image before sending it to the server, thus, reducing server load + logic.

Usually I also do that for the image to be lighter, so do you plan a compress utility too? Something that could just compress the images size by downgrading quality or something maybe?

Thanks!

**Comment by Norserium** [comment]
@darkylmnx, actually, I don't understand your question completely.

It's the cropper library, so it should just give an user the possibility to choose the cropped area and crop it. It revolves around this process.

If you need to manipulate an image directly you should use another libraries and approaches. This library provides only the basic approaches for that.

For example:
1. You can [resize the result size](https://norserium.github.io/vue-advanced-cropper/guides/recipes.html#resize-the-result) to specific dimensions.
2. You can set the specific quality of an image while you get that image from a canvas:
```js
canvas.toBlob(blob => {
   /* callback */
}, 'image/jpeg', quality)
```

If you describe your use-case more thoroughly, perhaps I will be able to give you any other advice.
